cypress/e2e/home-spec.cy.js

```diff
@@ -0,0 +1,59 @@
+describe('rancid tomatillos homepage user flows', () => {
+
+  beforeEach(() => {
+    cy.intercept('GET', 'https://rancid-tomatillos.herokuapp.com/api/v2/movies', {
+      statusCode: 200,
+      fixture: '.././fixtures/all-movies-test.json'
+    })
+    .visit('http://localhost:3000');
+  });
+  
+  it('should display all navbar elements upon navigating to the app', () => {
+    cy.contains('h1', 'RANCID')
+    cy.contains('h2', 'TOMATILLOS')
```

Comment:
If Cypress can do this.

For tests, they dont care about the type case, just that they exist. During development we can change type case and then the tests break.

Our team uses regex instead.

```suggestion
    cy.contains('h1', /rancid/i)
    cy.contains('h2', /tomatillos/i)
```